src/index.js

```diff
@@ -24,10 +24,12 @@ export default function (babel) {
           return
         }
         const {id: {name}} = node
-        const {referencePaths} = path.scope.getBinding(name)
-        referencePaths.forEach(reference => {
-          identifiers.add(reference)
-        })
+        if(path.scope.getBinding(name)){
```

Comment:
Could we do:

```js
const binding = path.scope.getBinding(name)
if (binding) {
  const {referencePaths} = binding
  // ...etc
}
```

Also, with this fix, are the display names being set correctly still?